nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -104,6 +111,21 @@
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
+    public static final PropertyDescriptor HEALTH_CHECK_PORT = new PropertyDescriptor.Builder()
+            .name("health-check-port")
+            .displayName("Listening Port for health check requests")
```

Comment:
Property names should be capitalized as titles: Listening Port for Health Check Requests